### PR TITLE
Add README for localization package

### PR DESCRIPTION
**Pull request by BigRoma**

### CODE REVIEW

File: packages/localization/README-ru.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+[English](./README.md)
+
+# Локализация проекта [Quiver UI](https://github.com/tonlabs/UIKit)
+[![PR's приветствуются](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+
+В этом репозитории размещены файлы локализации проекта Quiver UI.
```
Comment (sertony) [inline packages/localization/README-ru.md]: No just Quiver UI, but all the components of UIKit.

**Review by savelichalex — APPROVED** [review]

File: packages/localization/README-ru.md (hunk 1 above)
Comment (BigRoma) [reply packages/localization/README-ru.md]:
```suggestion
В этом репозитории размещены файлы локализации проекта UIKit. 
```

File: packages/localization/README-ru.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+[English](./README.md)
+
+# Локализация проекта [Quiver UI](https://github.com/tonlabs/UIKit)
```
Comment (BigRoma) [inline packages/localization/README-ru.md]:
```suggestion
# Локализация проекта [UIKit](https://github.com/tonlabs/UIKit)
```

**Review by savelichalex — APPROVED** [review]

**Review by savelichalex — APPROVED** [review]

**Comment by savelichalex** [comment]
@SAnatoliiS can we merge it?

**Review by SAnatoliiS — APPROVED** [review]

**Pull request merged**